.github/workflows/build-linux.yml

```diff
@@ -19,7 +19,7 @@ jobs:
             platform: linux/amd64
           - arch: ia32
             runner: ubuntu-latest
-            platform: linux/386
+            platform: linux/amd64
```

Comment:
Official dart docker image do not have `linux/386` support. Use `linux/amd64` to build snapshot, as ia32 has no AOT support anyways.